Update to Electron 32 fails

<!-- For Bugs only

For Questions, Support, Ideas, please use:

https://github.com/deepnest-io/Deepnest/discussions
-->

**Describe the bug**
Updating to electron `32.2.7` fails with errors like

```
C:\Users\abebeos\.electron-gyp\32.2.7\include\node\v8-internal.h(109,67): warning C4003: not enough arguments for function-like macro invocation 'max' [P:\work\Deepnest\build\addon.vcxproj]
  (compiling source file '../src/addon.cc')

```




**To Reproduce**

change electron to     "electron": "32.2.7" (within package.json)

```
npm run dist-all
```

**Desktop (please complete the following information):**

windows 10

### Possible Solution

* https://github.com/electron/electron/issues/40760#issuecomment-1979048889

--- a/binding.gyp
+++ b/binding.gyp
@@ -14,6 +14,9 @@
             'cflags!': [ '-fno-exceptions', "-m64" ],
             "ldflags": [ "-m elf_i386" ],
             'cflags_cc!': [ '-fno-exceptions', '-fPIC -m64' ],
+            "defines": [
+              "NOMINMAX"
+            ],
           }
         ],
         [ 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "deepnest",
-  "version": "1.2.1",
+  "version": "1.2.2",
   "description": "Deep nesting for Laser and CNC",
   "main": "main.js",
   "license": "MIT",
@@ -30,7 +30,7 @@
     "@electron/packager": "^18.3.6",
     "@electron/rebuild": "^3.2.9",
     "@types/node": "^22.10.1",
-    "electron": "31.7.6",
+    "electron": "32.2.7",
     "nan": "^2.20.0",
     "shx": "^0.3.4"
   },

